lisa/feature.py

```diff
@@ -67,3 +67,9 @@ def __getitem__(self, feature_type: Type[T_FEATURE]) -> T_FEATURE:
 
         assert feature
         return cast(T_FEATURE, feature)
+
+    def _is_supported(self, feature_type: Type[T_FEATURE]) -> bool:
+        if self._supported_features.get(feature_type.name()):
```

Comment:
`get` may throw exception, if it doesn't exist. Try `name in self._supported_features`

Reply:
And it can be one line like:
`return feature_type.name() in self._supported_features`